aten/src/ATen/native/cpu/ReduceUtils.h

```diff
@@ -105,7 +107,15 @@ inline Vectorized<scalar_t> _max(const Vectorized<scalar_t>& x, const Vectorized
 }
 
 template <typename scalar_t>
-inline scalar_t _min(const scalar_t& x, const scalar_t& y) {
+inline typename std::enable_if<std::is_same<scalar_t, Vec2>::value, Vec2>::type
```

Comment:
is `scalar_t` referers to vectorized type ? if so, use `vec_t` which is more meaningful.